### PR TITLE
Sign up citizen UI: Part 1

### PR DESCRIPTION
**Pull request by jonathanchuangcodes**
## Dev Summary

Putting this up with work left to do to keep the PR size reasonable.

- added login page image assets
- added some theme defaults
- lots of formatting changes to citizen sign up
- got rid of steppers

### CODE REVIEW

**Comment by jd2rogers2** [comment]
@jonathanchuangcodes re: your standup comments (from last week 😞 sorry i'm late on this reply)
- back buttons: just add em in yourself for now, maybe just grouped with the other skip and next buttons, so in line and right aligned with them
  - but also, side note, i don't think we want a skip button on the location page at least
- re: https://react-svgr.com/ go for it and give it a try!!

File: client/src/assets/sign-up-step-1.svg
Comment (esteban-gs) [inline client/src/assets/sign-up-step-1.svg:1]: We could benefit from suffixing the image name with the page or cta they go with, for example `client/src/assets/sign-up-contact-info-step.svg`

File: client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx [hunk 1]
```diff
@@ -181,136 +188,199 @@ export const SignUpUserAndNonprofit = () => {
...
           </Grid>
           <Grid item sx={{ marginBottom: '30px' }}>
-            <Typography component="p" align="left" gutterBottom>
+            <Typography
+              component="p"
+              align="left"
+              gutterBottom
+              fontSize={'15px'}
+              marginTop={'10px'}
+            >
```
Comment (esteban-gs) [inline client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx:203]: Should we integrate with a theme variant?

File: client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx [hunk 2]
```diff
@@ -181,136 +188,199 @@ export const SignUpUserAndNonprofit = () => {
...
+                          <Typography sx={{ fontSize: '16px', marginBottom: '10px' }}>
+                            State
+                          </Typography>
+                          <TextField
+                            {...field}
+                            fullWidth
+                            label="Add location data here"
+                            onKeyUp={() => {
+                              if (!errors.state) {
+                                // setTriggerEinSearch(true);
```
Comment (esteban-gs) [inline client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx:294]: If not used, let's remove this comment

File: client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx [hunk 3]
```diff
@@ -181,136 +188,199 @@ export const SignUpUserAndNonprofit = () => {
...
+                          <Typography sx={{ fontSize: '16px', marginBottom: '10px' }}>
+                            IRS Nonprofit Organization Classification
+                          </Typography>
+                          <TextField
+                            hiddenLabel
+                            {...field}
+                            fullWidth
+                            onKeyUp={() => {
+                              if (!errors.nonprofit_classification) {
+                                // setTriggerEinSearch(true);
```
Comment (esteban-gs) [inline client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx:377]: Remove comment if unused

File: client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx [hunk 4]
```diff
@@ -604,7 +674,7 @@ export const SignUpUserAndNonprofit = () => {
                       variant="contained"
                       disabled={activeStep === 0}
                       onClick={handleBack}
-                      sx={{ mr: 1 }}
+                      sx={{ mr: 1, display: `${activeStep === 0 && 'none'}` }}
```
Comment (esteban-gs) [inline client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx:677]: Will this still work now that the multi-step component has been removed?

File: client/src/theme/components.ts [hunk 5]
```diff
@@ -1,6 +1,25 @@
...
+  MuiButton: {
+    styleOverrides: {
+      root: {
+        border: '1px solid #323232',
+        borderRadius: '10px',
+        fontSize: '16px',
+        fontWeight: '600',
+        lineHeight: '24px',
+        padding: '8px, 10px, 8px, 20px',
+        color: '#323232',
```
Comment (esteban-gs) [inline client/src/theme/components.ts:16]: Nice!

**Review by esteban-gs — APPROVED** [review]
Looking good! Thanks for putting this together. A few comments most are non-blocking

File: client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx (hunk 4 above)
Comment (jonathanchuangcodes) [reply client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx:677]: yep!

**Pull request merged**